.github/workflows/wheel-builder.yml

```diff
@@ -116,3 +116,24 @@ jobs:
         with:
           name: "bcrypt-${{ github.event.client_payload.BUILD_VERSION }}-${{ matrix.WINDOWS.WINDOWS }}-${{ matrix.PYTHON.VERSION }}"
           path: bcrypt-wheelhouse\
+
+  manylinux-arm64:
+     name: "Python ${{ matrix.PYTHON }} for manylinux2014-aarch64"
+     runs-on: ubuntu-latest
+     strategy:
+       matrix:
+         PYTHON: ["cp36-cp36m"]
+       fail-fast: false
+     steps:
+     - uses: actions/checkout@v2
+     - run: |
+         docker run --rm --privileged hypriot/qemu-register
+     - uses: docker://quay.io/pypa/manylinux2014_aarch64
+       with:
+        args: 'bash .github/scripts/arm64_create_wheels.sh ${{ matrix.PYTHON }}'
```

Comment:
Is it not possible to put the shell script here? Using the pipe `|` and then having all the shell script on indented lines below it should work right?

Reply:
Thanks for the feedback.
I have checked without the script as well, we have to write the steps as below:
```
- uses: docker://quay.io/pypa/manylinux2014_aarch64
  with: 
     args: |
           bash -c "/opt/python/${{ matrix.PYTHON }}/bin/pip install virtualenv;
                    /opt/python/${{ matrix.PYTHON }}/bin/python -m virtualenv .venv;
                    .venv/bin/pip install setuptools wheel cffi six;
                    REGEX='cp3([0-9])*';
                    if [[ '${{ matrix.PYTHON }}' =~ $REGEX ]]; then PY_LIMITED_API='--py-limited api=cp3${BASH_REMATCH[1]}'; fi;
                    .venv/bin/python setup.py bdist_wheel $PY_LIMITED_API -d tmpwheelhouse;
                    auditwheel repair tmpwheelhouse/bcrypt*.whl -w wheelhouse/;
                    .venv/bin/pip install bcrypt --no-index -f wheelhouse/;"
```

If you approve of the above changes, I will amend the same in the commit.

Reply:
Looks good assuming the quoting is safe. Have you tested on your own branch?

Reply:
Updated and amended the same. I have verified also in my test branch and it is building successfully. Please check [report logs](https://github.com/odidev/bcrypt/runs/935325463?check_suite_focus=true) for more details.